core/services/log/registrations.go

```diff
@@ -291,9 +299,19 @@ func (r *subscribers) sendLog(log types.Log, latestHead eth.Head,
 			}
 		}
 
+		jobID := listener.JobID()
+		if !exists {
+			// Create unconsumed broadcast
+			if err := bc.CreateBroadcast(log.BlockHash, log.BlockNumber, log.Index, jobID); err != nil {
+				logger.Errorw("Could not create broadcast log", "blockNumber", log.BlockNumber,
+					"blockHash", log.BlockHash, "address", log.Address, "jobID", jobID, "error", err)
+				continue
+			}
+		}
```

Comment:
Ensure that an unconsumed entry exists before handing off to the listener below and potentially dropping from the pool.